template/README.md

```diff
@@ -511,6 +512,39 @@ To override this, specify the `homepage` in your `package.json`, for example:
 
 This will let Create React App correctly infer the root path to use in the generated HTML file.
 
+## Netlify
+
+**To do a manual deploy to Netlify's CDN:**
+
+```sh
+npm install netlify-cli
+netlify deploy
+```
+
+Choose `build` as the path to deploy.
+
+**To setup continuous delivery**
+
+With this setup Netlify will build and deploy when you push to git or open a pull request:
+
+1. [Start a new netlify project](https://app.netlify.com/signup)
+2. Pick your Git hosting service and select your create-react-app repository
+3. Click `Build your site`
+
+**Support for client site routing:**
+
+To support `pushState` add a `src/_redirects` file with the following rewrite rule:
+
+```
+/*  /index.html  200
+```
+
+Then add this import statement to `src/index.js`:
+
+```js
+import "file?name=_redirects!./_redirects";
```

Comment:
Well that was the whole reason for calling it `static`, since people normally use `public` folders for images and CSS that should actually go into `src` in our case. Thus, `static`!